package.json

```diff
@@ -37,35 +37,35 @@
     "not ie > 0"
   ],
   "devDependencies": {
-    "@babel/cli": "^7.22.5",
-    "@babel/core": "^7.22.5",
-    "@babel/eslint-parser": "^7.22.5",
-    "@babel/preset-react": "^7.22.5",
-    "@babel/runtime": "^7.22.5",
-    "@cfpb/cfpb-atomic-component": "0.27.0",
-    "@cfpb/cfpb-buttons": "0.27.0",
-    "@cfpb/cfpb-core": "0.27.0",
-    "@cfpb/cfpb-design-system": "^0.27.1",
-    "@cfpb/cfpb-expandables": "^0.17.12",
-    "@cfpb/cfpb-forms": "0.27.0",
-    "@cfpb/cfpb-grid": "0.27.0",
-    "@cfpb/cfpb-icons": "^0.31.0",
-    "@cfpb/cfpb-layout": "0.27.1",
-    "@cfpb/cfpb-notifications": "0.27.0",
-    "@cfpb/cfpb-pagination": "0.27.0",
-    "@cfpb/cfpb-tables": "0.27.0",
-    "@cfpb/cfpb-typography": "0.27.0",
+    "@babel/cli": "^7.23.9",
+    "@babel/core": "^7.23.9",
+    "@babel/eslint-parser": "^7.23.10",
+    "@babel/preset-react": "^7.23.3",
+    "@babel/runtime": "^7.23.9",
+    "@cfpb/cfpb-atomic-component": "0.37.0",
+    "@cfpb/cfpb-buttons": "0.37.0",
+    "@cfpb/cfpb-core": "0.37.0",
+    "@cfpb/cfpb-design-system": "^0.37.2",
+    "@cfpb/cfpb-expandables": "^0.37.0",
```

Comment:
it all looks the same, but is this issue still valid?  

https://github.com/cfpb/ccdb5-ui/issues/463

Reply:
Updated here. Hopefully I got everything https://github.com/cfpb/ccdb5-ui/pull/490/commits/96b01034f59d78277900dc11c19c0fb4f66da5e0

Before:
<img width="403" alt="Screenshot 2024-02-22 at 4 50 37 PM" src="https://github.com/cfpb/ccdb5-ui/assets/704760/6818e5cc-06fb-4b15-8453-e881fc877f82">

After:
<img width="395" alt="Screenshot 2024-02-22 at 4 50 47 PM" src="https://github.com/cfpb/ccdb5-ui/assets/704760/365185c4-0b22-4eed-af45-ea1bff6f7507">

Reply:
wow, that was fast.  Faster than me trying to figure it out!